website/example.css

```diff
@@ -0,0 +1,163 @@
+body, html {
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    font-family: Avenir Next;
+}
+
+* {
+    padding: 0;
+    margin: 0;
+    border: none;
+}
+
+*:focus {
+    outline: none;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+/* ------------------------------------------------------------------------------------------------------------------------ */
+
+#header {
```

Comment:
[Using IDs for styling and having more that one instance of an ID is a bad pattern](https://stackoverflow.com/questions/9721166/why-should-i-not-use-style-ids-using-css). The HTML spec assumes a single instance of any ID per page, the ID also has [a higher css specificity](https://css-tricks.com/specifics-on-css-specificity/) and can cause a lot of styling issues down stream.